src/main/kotlin/org/jetbrains/research/testspark/tools/evosuite/generation/EvoSuiteProcessManager.kt

```diff
@@ -84,7 +84,7 @@ class EvoSuiteProcessManager(
             if (ToolUtils.isProcessStopped(errorMonitor, indicator)) return null
 
             val regex = Regex("version \"(.*?)\"")
-            val version = regex.find(CommandLineRunner.run(arrayListOf(evoSuiteSettingsState.javaPath, "-version")))
+            val version = regex.find(CommandLineRunner.run(arrayListOf(evoSuiteSettingsState.javaPath, "-version")).second)
```

Comment:
Almost unreadable.

I recommend moving the following part into a named variable:
```kotlin
CommandLineRunner.run(arrayListOf(evoSuiteSettingsState.javaPath, "-version"))
```